raccoongang/edx-platform
# Disable csrf check. Deleted middelware for delete csrf token. Fix pro…

**Description:** 
Disable csrf check. Deleted middelware for delete csrf token. Fix problem with redirect - add sites params (disable global).
Deleted unused lines in view. Fix parse course id.

**Youtrack:** https://youtrack.raccoongang.com/issue/NRG-39

**Merge deadline:** List merge deadline (if any)

**Configuration instructions:** List any non-trivial configuration
instructions (if any).

**Testing instructions:**
 1. check out the creation and subscriptions
 2. check redirect from the registration page


**For OPS:** 
In admin pages  -  Site configurations (for current site) add next params: 

  "ENABLE_REDIRECT_REGISTER":true,
  "REGISTER_REDIRECT_URL":"http://www.nrgsustainablefuture.com/"
